docs/source/cli_help/anno.rst

```diff
@@ -0,0 +1,42 @@
+.. role:: bash(code)
+
+.. _anno:
+
+anno
+----
+Usage:
+
+.. code-block:: shell
+
+    git machete anno [-b|--branch=<branch>] [<annotation text>]
+    git machete anno -H|--sync-github-prs
+
+If invoked without any argument, prints out the custom annotation for the given branch (or current branch, if none specified with :bash:`-b/--branch`).
+
+If invoked with a single empty string argument, like:
+
+.. code-block:: shell
+
+    $ git machete anno ''
+
+then clears the annotation for the current branch (or a branch specified with :bash:`-b/--branch`).
+
+If invoked with :bash:`-H` or :bash:`--sync-github-prs`, annotates the branches based on their corresponding GitHub PR numbers and authors.
+Any existing annotations are overwritten for the branches that have an opened PR; annotations for the other branches remain untouched.
+
+To allow GitHub API access for private repositories (and also to correctly identify the current user, even in case of public repositories),
+a GitHub API token with :bash:`repo` scope is required, see :bash:`https://github.com/settings/tokens`. This will be resolved from the first of:
```

Comment:
Also, nit:
```suggestion
a GitHub API token with ``repo`` (or just ``repo:public``) scope is required, see :bash:`https://github.com/settings/tokens`. This will be resolved from the first of:
```